eskip/eskip.go

```diff
@@ -531,9 +533,13 @@ func applyPredicates(route *Route, proute *parsedRoute) error {
 		default:
 			route.Predicates = append(route.Predicates, p)
 		}
+
+		if err != nil {
+			return fmt.Errorf("invalid route %q: %w", proute.id, err)
+		}
 	}
 
-	return err
+	return nil
```

Comment:
Moved error check inside of the loop to add route id. 

Its also as better fix then #170 was for #169